src/Products/Products.Core/Features/Categories/Queries/GetCategory.cs

```diff
@@ -0,0 +1,43 @@
+﻿using IGroceryStore.Products.Core.Exceptions;
+using IGroceryStore.Products.Core.Persistence.Contexts;
+using IGroceryStore.Products.Core.ReadModels;
+using IGroceryStore.Shared.Abstraction.Common;
+using IGroceryStore.Shared.Abstraction.Constants;
+using IGroceryStore.Shared.Abstraction.Queries;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.EntityFrameworkCore;
+
+namespace IGroceryStore.Products.Core.Features.Categories.Queries;
+
+internal record GetCategory(ulong Id) : IHttpQuery;
```

Comment:
I don't see use case for get only one category 
get categories with paged result make more sense